modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oauth2/OAuth2RoleClaimTestCase.java

```diff
@@ -23,89 +23,103 @@
 import java.util.List;
 
 public class OAuth2RoleClaimTestCase extends OAuth2ServiceAbstractIntegrationTest {
 
-    private static final String OAUTH_ROLE = "Internal/oauthRole";
-    private static final String ROLES_CLAIM_URI = "http://wso2.org/claims/roles";
+    private static final String OAUTH_ROLE = "oauthRole";
+    private static final String USERS_PATH = "users";
     private static final String OIDC_ROLES_CLAIM_URI = "roles";
-    private static final String FIRST_NAME_VALUE = "FirstName";
-    private static final String LAST_NAME_VALUE = "LastName";
-    private static final String EMAIL_VALUE = "email@wso2.com";
-    private static final String OPENID_SCOPE_PROPERTY = "openid";
-    private static final String OPENID_SCOPE_RESOURCE = "/_system/config/oidc";
-    private static final String MULTI_ATTRIBUTE_SEPARATOR = ",";
+    private static final String GIVEN_NAME = "testUser";
+    private static final String FAMILY_NAME = "test";
+    private static final String HOME_ATTRIBUTE = "home";
+    private static final String HOME_EMAIL = "testuser11@gmail.com";
+    private static final String WORK_ATTRIBUTE = "work";
+    private static final String WORK_EMAIL = "testuser99@wso2.com";
+    private static final String USER_USERNAME = "testuser99";
+    private static final String USER_PASSWORD = "testuser@123";
+    private static final String EMPLOYEE_NUMBER = "Abc123";
+    private static final String MANAGER_NAME = "wso2TestManage";
+
 
     private String consumerKey;
     private String consumerSecret;
+    private String applicationId;
+    private String roleId;
+    private String userId;
 
-    private DefaultHttpClient client;
-
-    private static final String FIRST_NAME_CLAIM_URI = "http://wso2.org/claims/givenname";
-    private static final String LAST_NAME_CLAIM_URI = "http://wso2.org/claims/lastname";
+    private CloseableHttpClient client;
+    private SCIM2RestClient scim2RestClient;
 
-    private static final String USERNAME = "oauthuser";
-    private static final String PASSWORD = "oauthuser";
+    private String USERNAME;
+    private String PASSWORD;
 
     @BeforeClass(alwaysRun = true)
     public void testInit() throws Exception {
 
         super.init(TestUserMode.TENANT_USER);
 
+        this.USERNAME = tenantInfo.getContextUser().getUserName();
+        this.PASSWORD = tenantInfo.getContextUser().getPassword();
         setSystemproperties();
-        client = new DefaultHttpClient();
+        client = HttpClients.createDefault();
+        scim2RestClient = new SCIM2RestClient(serverURL, tenantInfo);
 
-        remoteUSMServiceClient.addRole(OAUTH_ROLE, null, null);
-        remoteUSMServiceClient.addUser(USERNAME, PASSWORD, null, getUserClaims(), "default",
-                false);
+        roleId = scim2RestClient.addRole(getRoleCreationInfo());
+        userId = scim2RestClient.createUser(getUserCreationInfo());
     }
 
     @AfterClass(alwaysRun = true)
     public void atEnd() throws Exception {
-
```

Comment:
Shall we fix the formatting issue by adding the missing new line?

Reply:
Fixed with https://github.com/wso2/product-is/pull/20096